src/main/java/com/geronimo/model/AbstractEntity.java

```diff
@@ -0,0 +1,13 @@
+package com.geronimo.model;
+
+import lombok.Setter;
+
+import javax.persistence.MappedSuperclass;
+
+@Setter
+@MappedSuperclass
+public class AbstractEntity {
+
+    @javax.persistence.Version
```

Comment:
Minor things, but makes code look better, you can add import javax.persistence.Version and use Version annotation without explicit package name.